### PR TITLE
Return recent bills voted on by chamber with legislators.

### PR DESCRIPTION
**Pull request by primitive-type**
@jimbol This PR nearly completes issue #4. I could use a bit of help with how to cache the results in memcached though, using the caching library we're using.

### CODE REVIEW

File: src/controllers/legislator.js [hunk 1]
```diff
@@ -197,6 +204,26 @@ Legislator.prototype.getContributors = function(legislator, callback){
...
+
+  bill.findByChamber({
+    chamber: legislator.chamber,
+  }, function(response) {
+
+    _this.bills = _this.bills.concat( response.bills );
+
+    response.bills.map(function(item) {
+
+      item.bill_id += legislator.entityId;
```
Comment (jimbol) [inline src/controllers/legislator.js]: Why are we combining `id`s here?

File: src/models/bill.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+// Constructor
+function Bill(){
+  this.query = {
+    apikey: config.apiKey
+  };
+  this.endpoint = '/bills';
+  this.url = config.urls.congress;
+}
+
+Bill.prototype = new Model();
```
Comment (jimbol) [inline src/models/bill.js]: Could we convert `Bill` to a iojs class?

File: src/models/bill.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+
+Bill.prototype.findById = function(query, callback) {
+  this.url += query.id;
+  this.find(query, callback);
+};
+
+Bill.prototype.findByChamber = function(query, callback){
+
+  if (query.chamber === 'senate') {
+      query['history.senate_passage_result__exists'] = true;
```
Comment (jimbol) [inline src/models/bill.js]: These query params are just hilarious.  Ohhh Sunlight...

**Comment by jimbol** [comment]
This looks pretty good, just a couple questions.

**Comment by primitive-type** [comment]
@jimbol Oops, I forgot to use an ES6 class for the bill model because I hadn't merged in upstream changes to my fork of the project yet. So all of my code still had the old prototype code. I've merged in the latest changes from master and converted the bill model to use a class. When I first merged in the changes, though, I started getting this strange error and I can't figure out what's causing it:

```ReferenceError: this is not defined
    at Function.Legislator (/www/sites/onbehalf/on-behalf-api/src/models/legislator.js:9:5)```

It seems like something wrong with the Legislator model class, but it's such a simple class definition, I can't think of what it could be. Maybe we can look into this a bit when we meet up.

File: src/controllers/legislator.js [hunk 4]
```diff
@@ -147,7 +150,10 @@ Legislator.prototype.getDependencies = function(responseData, callback){
         }],
         getIndustries: ['getEntity', function(callback){
           _this.getIndustries(legislator, callback);
-        }]
+        }],
+        getBills: ['getEntity', function(callback){
+        _this.getVotedBills(legislator, callback);
```
Comment (jimbol) [inline src/controllers/legislator.js]: non blocker, this isnt indented...

File: src/models/bill.js [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+      this.find( query, callback );
+    }
+
+    formatResponse(body) {
+      var responseData = {};
+      responseData = JSON.parse( body ).results;
+
+      responseData = responseData.filter(function(item) {
+        // `undefined != null` coerces undefined to null
+        return item.bill_id != null;
```
Comment (jimbol) [inline src/models/bill.js]: We probably dont need this.  Must be left overs from `contributors.js`.

**Pull request merged**